<launch>
    <node pkg="srrc_simple_obj_detector" type="test_simple_detector_node" name="test_simple_detector_node" output="screen">
	<param name="input_image_topic" value="/turret_stereo/left/image" />
	<param name="input_image_transfer_hint" value="compressed" />

	<param name="saturation_threshold" value="168" />
	<param name="brightness_threshold" value="20" />
	<param name="use_bw_result" value="false" />

        <!-- reset unused -->
	<param name="output_folder" value="" />
	<param name="show_images" value="false" />
	<param name="src_image_src_folder" value="" />
	<param name="src_image_file" value="" />
    </node>
</launch>

